downstream/modules/platform/proc-create-a-user.adoc

```diff
@@ -0,0 +1,52 @@
+[id="proc-create-a-user_{context}"]
+
+= Creating a User
+
+This procedure creates a user that can be used to log in to {HubName}.
+As the `hubadmin` role is assigned, this user has Super Administration privileges when logged in.
+
+Use the following procedure to create a Keycloak User.
```

Comment:
With the `.Procedure` title I don't think lead-in sentences like this add anything. It's just fluff. Suggest removing it for minimalism.